OneCommunityGlobal/HighestGoodNetworkApp
# Eduardo make reports page and all reports look as good as possible on phones

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/b335b8cc-fc9e-4cf1-b249-e3d2ef2b0332)
## Implements Mobile Reports Page
Make Reports Page looks good as possible on phones.

## Related PRS:
Previous PR addressing this feature [PR#865](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/865)

## Main changes explained:
- Added Media Queries to adjust the style deppending on the screen's width
- Created `classNames` for some components to implement the feature
- Adjusted overlapping style of Reports Page Components

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as owner/admin user
4. go to Reports → Reports → Right click to Inspect the Page (F12) → Toogle Device Tool Bar (Ctrl + Shift + M)
5. select a pre-defined width (Tablet - 768px / Mobile L - 425px / Mobile M - 375px / Mobile S - 320px)
6. click on each card (Projects/People/Teams) → Scroll Down → Pick a Project/Person/Team
7. verify if the important/relevant information are "looking good" on phones.

## Video of changes (Turn sound On to hear my disclaimer on how I addressed this feature)

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/40707b21-e4d3-4c81-9d4c-4e4d6ae18d2c



## Screenshots of changes:

![People Report](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/704a53c3-9058-42ab-b4e8-2b41bdfb16e7)
![Reports Page](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/02f5c4eb-c837-44b3-8d3d-08cf20bc7de2)
![Team Report](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/7cec824c-7db4-4645-ba0c-0bae47d83dae)



## Note:
About the "Tasks Table Total: 0" on the bottom of the page when accessing Reports -> Projects -> Pick Any
This part of the app is not yet finished.
So what you are seeing is it's expected behaviour for now.
There will be another PR addressing this feature/style but it's not this one.

If you find any issue, please follow these steps first:

1. Close Inspect Window (F12) with Toogle Device Bar On (Ctrl + Shift + M)
2. Reload the page (Ctrl + R)
3. Re-open Inspect Window (F12)
4. Reload the page once more with Toogle Device Bar On (Ctrl + Shift + M)
5. Check if the issue is still persisting

## Video of expected behaviour (Turn sound On)
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/8919d256-ddef-4351-86b4-dcba21bb5c71
